Parsers/ASimProcessEvent/Parsers/ASimProcessCreateTrendMicroVisionOne.yaml

```diff
@@ -0,0 +1,156 @@
+          and detail_eventSubId_s == "TELEMETRY_PROCESS_CREATE"
+      | parse filters_s with * "[" filters: string "]"
+      | parse-kv filters as (description: string, name: string) with (pair_delimiter=",", kv_delimiter=":", quote='"')
+      | extend
+          ActingProcessId = tostring(toint(detail_processPid_d)),
+          TargetProcessId = tostring(toint(detail_objectPid_d)),
+          ParentProcessId = tostring(toint(detail_parentPid_d)),
+          TargetProcessCreationTime = unixtime_milliseconds_todatetime(detail_objectLaunchTime_d),
+          ActingProcessCreationTime = unixtime_milliseconds_todatetime(detail_processLaunchTime_d),
+          ActingProcessFilename = GetFilenamePart(detail_processFilePath_s),
+          ParentProcessCreationTime = unixtime_milliseconds_todatetime(detail_parentLaunchTime_d),
+          ParentProcessName = detail_parentName_s,
+          TargetProcessFilename = GetFilenamePart(detail_objectFilePath_s),
+          ActingProcessFileSize = tolong(detail_processFileSize_d),
+          TargetUserSessionId = tostring(toint(detail_objectAuthId_d)),
+          ActorSessionId = tostring(toint(detail_authId_d)),
+          TargetProcessMD5 = replace_string(detail_objectFileHashMd5_g, "-", ""),
+          ActingProcessMD5 = replace_string(detail_processFileHashMd5_g, "-", ""),
+          ParentProcessMD5 = replace_string(detail_parentFileHashMd5_g, "-", ""),
+          AdditionalFields = bag_pack(
+                        "description",
+                        description,
+                        "name",
+                        name,
+                        "tags",
+                        detail_tags_s
+                    )
+      | lookup EventSeverityLookup on detail_filterRiskLevel_s
+      | invoke _ASIM_ResolveDvcFQDN('detail_endpointHostName_s')
+      | lookup IntegrityLevelLookup on $left.detail_parentIntegrityLevel_d == $right.IntegrityLevel
+      | project-rename ParentProcessIntegrityLevel = IntegrityType
+      | lookup IntegrityLevelLookup on $left.detail_objectIntegrityLevel_d == $right.IntegrityLevel
+      | project-rename TargetProcessIntegrityLevel = IntegrityType
+      | lookup IntegrityLevelLookup on $left.detail_integrityLevel_d == $right.IntegrityLevel
+      | project-rename ActingProcessIntegrityLevel = IntegrityType
+      | extend
+          EventCount = int(1),
+          EventProduct = "Vision One",
+          EventResult = "Success",
+          EventSchemaVersion = "0.1.4",
+          EventType = "ProcessCreated",
+          EventVendor = "Trend Micro",
+          EventSchema = "ProcessEvent",
+          DvcAction = "Allowed"
+      | project-rename
+          ActorUsername = detail_processUser_s,
+          EventStartTime = detail_eventTimeDT_t,
+          TargetProcessCommandLine = detail_objectCmd_s,
+          TargetProcessName = detail_objectName_s,
+          TargetUsername = detail_objectUser_s,
+          ActingProcessCommandLine = detail_processCmd_s,
```

Comment:
sometimes we are having double quotes, can we remove them?

Reply:
Done